building_dialouge_webapp/heat/views.py

```diff
@@ -228,9 +230,136 @@ class Results(SidebarNavigationMixin, TemplateView):
+        html_scenario2 = generate_html_table(df_scenario2_investments, df_scenario2_subsidies, df_scenario2_sum, "tab_scenario2")
+
+        # Tab-Steuerelemente
+        tabs_html = """<div class='tabs'>
+            <button class='tab-button' style="background-color: #1b9e77;" onclick="showTab('tab_scenario1', '#1b9e77')">Szenario 1</button>
+            <button class='tab-button' style="background-color: #7570b3;" onclick="showTab('tab_scenario2', '#7570b3')">Szenario 2</button>
+        </div>"""
+
+        # Zusammenfügen
+        full_html = f"""<style>
+            .tabs {{ margin-bottom: 0; position: relative; top: 0; left: 0; }}
+            .tab-button {{ margin-right: 5px; padding: 10px; color: white; border: none; cursor: pointer; }}
+            .tab-button:hover {{ opacity: 0.9; }}
+            .table {{ display: none; width: 100%; margin: 0; border-collapse: collapse; }}
+            .table th, .table td {{ border: 1px solid lightgrey; padding: 10px; text-align: left; }}
+            .table th {{ width: 70%; border-right: none; border-left: none; background-color: #f9f9f9; }}
+            .tab_scenario1 {{ display: table; border: 10px solid #1b9e77; border-radius: 10px; }}
+            .tab_scenario2 {{ border: 10px solid #7570b3; border-radius: 10px; }}
+        </style>
+        <script>
+            function showTab(tabName, borderColor) {{
+                var tables = document.querySelectorAll('.table');
+                tables.forEach(function(table) {{
+                    table.style.display = 'none';
+                    table.style.border = '10px solid ' + borderColor;
+                    table.style.borderRadius = '10px';
+                }});
+                document.querySelectorAll('.' + tabName).forEach(function(table) {{
+                    table.style.display = 'table';
+                }});
+            }}
+        </script>
+        {tabs_html}
+        {html_scenario1}
+        {html_scenario2}
+        """
+
+        tabledata = {
+            'Maßnahme': ['Heiztechnologie wechseln', 'Fassade sanieren', 'Dach sanieren', 'Fenster austauschen',
+                         'Kellerdecke dämmen'],
+            'Szenario 1': [-50, -50, 'nicht ausgewählt', 'nicht ausgewählt', 'nicht ausgewählt'],
+            'Szenario 2': [-88, -50, -45, 'nicht ausgewählt', 'nicht ausgewählt']
+        }
+        df = pd.DataFrame(tabledata)
+
+        # Generierte zweite Tabelle
+        #measures_table = style_table(df)
+
+        # Kontext hinzufügen
+        #context['measures_table'] = measures_table
+        # Kontext hinzufügen
+        context['html_content'] = full_html
+        context['hectare_scenario1'] = 2.2
+        context['hectare_scenario2'] = 1.3
+        return context
 
 class NextSteps(SidebarNavigationMixin, TemplateView):
     template_name = "pages/next_steps.html"
     extra_context = {
         "back_url": "heat:results",
     }
+
+
+def style_table(df):
```

Comment:
this should not be needed - if you define CSS classes you could move this to static CSS. Or do I miss something?